.docker/Dockerfile

```diff
@@ -97,6 +108,21 @@ RUN sudo apt-get -q update \
     && sudo apt-get clean -y \
     && sudo rm -rf /var/lib/apt/lists/*
 
+# Manually install MAVROS from source in the ws_blue/ workspace
+WORKDIR $USER_WORKSPACE/src/
+ARG MAVROS_RELEASE=ros2
+ARG MAVLINK_RELEASE=release/rolling/mavlink
+RUN git clone --depth 1 -b ${MAVROS_RELEASE} https://github.com/mavlink/mavros.git
+RUN git clone --depth 1 --recursive -b ${MAVLINK_RELEASE} https://github.com/mavlink/mavlink-gbp-release.git mavlink
+# mavgen uses future.standard_library for backwards compatibility with Python2;
+# However, this caused issues Python3.12 as installed in "noble".
+# Comment those lines out in mavlink.
+# n.b. This may not be the best way to do this.  A sed one-liner?
+RUN patch -d mavlink -p 1 < $USER_WORKSPACE/src/blue/.docker/patches/mavlink_standard_library.patch
```

Comment:
Applying a patch seems fine, but adds some extra files to maintain (and adds more required changes when we eventually install the binary). Here is the sed command for this patch:
```bash
sed -i -e 's/^from future import standard_library/#from future import standard_library/' \
          -e 's/standard_library.install_aliases()/#standard_library.install_aliases()/' \
          <file-name>
```
Not sure what the filename is here, so that would need to be fixed.

Reply:
Done!